### PR TITLE
AT_02.03.01_Envelope icon is visible

### PR DESCRIPTION
**Pull request by Albina-star**

### CODE REVIEW

File: cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_HeaderEnvelopeIcon.cy.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/// <reference types="Cypress" />
```
Comment (elviraKh1) [inline cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_HeaderEnvelopeIcon.cy.js:1]: file name should have name like US_02.03_EnvelopeIconUIAndFunc or US_02.03_EnvelopeIconUIAndFunctionality

File: cypress/pageObjects/HelpdeskPage.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 class HelpdeskPage {
```
Comment (elviraKh1) [inline cypress/pageObjects/HelpdeskPage.js:1]: this file should not be in the pr

File: cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_HeaderEnvelopeIcon.cy.js (hunk 1 above)
Comment (Albina-star) [reply cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_HeaderEnvelopeIcon.cy.js:1]: ok

**Review by elviraKh1 — APPROVED** [review]

**Review by elviraKh1 — APPROVED** [review]

File: cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_EnvelopeIconUIAndFunctionality.cy.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+
+describe('US_02.03 Header Envelope icon UI and functionality', () => {
+    const AGENT = Cypress.env('agent');
+
+    
+    beforeEach(() => {
+        cy.visit('/');
+        cy.login(AGENT.email, AGENT.password);
+
```
Comment (Rom-T) [inline cypress/e2e/testUiAndFunction/header/envelopeIconSection/US_02.03_EnvelopeIconUIAndFunctionality.cy.js:15]: Delete empty line

File: cypress/fixtures/helpdeskPage/headers.json [hunk 4]
```diff
@@ -1,3 +1,4 @@
 {
     "header": "value"
+
```
Comment (Rom-T) [inline cypress/fixtures/helpdeskPage/headers.json:3]: In this file, you just added an empty line. This does not apply to your PR. Please remove.

File: cypress/pageObjects/Header.js [hunk 5]
```diff
@@ -23,5 +24,9 @@ class Header {
     clickCloseBtn() {
         this.getCloseBtn().click();
     }
+
+    clickEnvelopeIcon() {
+        this.getEnvelopeIcon().click();
+    }
```
Comment (Rom-T) [inline cypress/pageObjects/Header.js:30]: The click method is not used anywhere. For the future, we do not add code. Please remove.

File: cypress/pageObjects/HelpdeskPage.js [hunk 6]
```diff
@@ -1,6 +0,0 @@
-class HelpdeskPage {
-    elements = {
-
-    }
-}
-export default HelpdeskPage;
```
Comment (Rom-T) [inline cypress/pageObjects/HelpdeskPage.js:6]: This class is a template that is in the structure of our project. Please restore.

**Pull request closed without merging**